I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/combat4.rpy", line 182, in script
    hide monster with moveoutbottem
  File "game/combat4.rpy", line 182, in <module>
    hide monster with moveoutbottem
NameError: name 'moveoutbottem' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/combat4.rpy", line 182, in script
    hide monster with moveoutbottem
  File "C:\renpy-8.2.1-sdk\renpy\ast.py", line 1359, in execute
    paired = renpy.python.py_eval(self.paired)
  File "C:\renpy-8.2.1-sdk\renpy\python.py", line 1209, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\renpy-8.2.1-sdk\renpy\python.py", line 1202, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/combat4.rpy", line 182, in <module>
    hide monster with moveoutbottem
NameError: name 'moveoutbottem' is not defined

Windows-10-10.0.22621 AMD64
Ren'Py 8.2.1.24030407
The Chosen One 1.3
Fri Aug 16 13:01:50 2024
